experimental/unicodeset_parser/src/parse.rs

```diff
@@ -1322,10 +1355,10 @@ pub fn parse(source: &str) -> Result<CodePointInversionListAndStringList<'static
 /// let my_set = parse("[abc]").unwrap();
 ///
 /// let mut variable_map = VariableMap::new();
-/// variable_map.insert("start".into(), 'a'.into());
-/// variable_map.insert("end".into(), 'z'.into());
-/// variable_map.insert("str".into(), "Hello World".into());
-/// variable_map.insert("the_set".into(), my_set.into());
+/// let _ = variable_map.insert_char("start".into(), 'a');
+/// let _ = variable_map.insert_char("end".into(), 'z');
+/// let _ = variable_map.insert_string("str".into(), "Hello World".into());
+/// let _ = variable_map.insert_set("the_set".into(), my_set);
```

Comment:
```suggestion
/// variable_map.insert_char("start".into(), 'a').unwrap();
/// variable_map.insert_char("end".into(), 'z').unwrap();
/// variable_map.insert_string("str".into(), "Hello World".into()).unwrap();
/// variable_map.insert_set("the_set".into(), my_set).unwrap();
///
/// // If a variable already exists, it will be replaced but `Err` is returned.
/// variable_map.insert_char("end".into(), 'Ω').unwrap_err();
```